### PR TITLE
Create KingTiger.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
create

### CODE REVIEW

File: KingTiger/KingTiger.fc [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+		Wait(4);
+
+          var HandlesVAR = Driver.WindowHandles;
+          foreach (var MyWindowVAR in HandlesVAR)
+          {
+        	    CheckForPopupsEnabled = false;
+           
+                Driver.SwitchTo().Window(MyWindowVAR);
+ 
+                if (Driver.Url.Contains("funfair.io"))
```
Comment (faucetcollector) [inline KingTiger/KingTiger.fc:253]: the login for kingtiger is on funfair?

File: KingTiger/KingTiger.fc [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+//=============//
+ public override int GetFaucetWaitTime()
+ {
+
+    if(ad)
+	{
+        ad=false; //solve the problem in start - after he return to script again
+        return GetWaitSetting();  
+
+        return GetWaitSetting();
```
Comment (faucetcollector) [inline KingTiger/KingTiger.fc:169]: duplicate

File: KingTiger/KingTiger.fc (hunk 1 above)
Comment (aviramdayan) [reply KingTiger/KingTiger.fc:253]: yeah all through the "funfair" platform

File: KingTiger/KingTiger.fc (hunk 2 above)
Comment (aviramdayan) [reply KingTiger/KingTiger.fc:169]: oh i'll send new commit

**Pull request merged**